azure_sdk/resources/deployment.py

```diff
@@ -43,13 +46,17 @@ def create_or_update(self, group_name, deployment_name, properties,
                          timeout):
         self.logger.info(
             "Create/Updating deployment...{0}".format(deployment_name))
-        resource_verify = bool(self.credentials.get('endpoint_verify', True))
+        resource_verify = bool(self.creds.get('endpoint_verify', True))
```

Comment:
Is this the only place where we reference `credentials` in the plugin?

Reply:
No, but all the other places look safe.